app/ui/src/app/common/ui-patternfly/syndesis-form-control.component.html

```diff
@@ -1,7 +1,7 @@
 <div [class.form-group]="formGroupClass"
      [class.has-error]="showErrorMessages"
      [formGroup]="group"
-     [ngClass]="[elementContainerClass, gridContainerClass]">
+     [ngClass]="[elementContainerClass, gridContainerClass, model?.inputType == 'hidden' ? 'hidden' : '']">
```

Comment:
I do not know how the other classNames operate here and what that `hidden` classNames doe,s but wether this component is styled as `inline`, you could use the more native-ish property `hidden`, like:

```html
 <div [class.form-group]="formGroupClass"
      [class.has-error]="showErrorMessages"
      [formGroup]="group"
      [hidden]="model?.inputType == 'hidden'">
      [ngClass]="[elementContainerClass, gridContainerClass">
```

Reply:
Do you know why it's failing CircleCI? Looks like the error is

> ERROR in src/app/common/ui-patternfly/syndesis-form-control.component.html(4,6): : Property 'inputType' does not exist on type 'DynamicFormControlModel'

Reply:
`DynamicFormControlModel` is a type native to the `ng-dynamic-forms/core` dependency. Perhaps there's been a bump to a new version where such property is not part of the `DynamicFormControlModel` class interface. Please double check wether you can see that property showing up as a class option in your IDE intellisense selectbox.

If not, please liaise with our `ng-dynamic-forms` expert @seanforyou23 since he might provide further details on what happened here. ;)

Reply:
@seanforyou23 do you know what's going on? `model.inputType` is showing up as an option in my IDE. 

![screen shot 2018-03-16 at 4 06 01 pm](https://user-images.githubusercontent.com/35148959/37544738-07ee4b0e-2934-11e8-8add-0fee33fd9241.png)